services/worker/tests/job_runners/split/test_duckdb_index.py

```diff
@@ -31,6 +31,7 @@
 from worker.job_runners.split.duckdb_index import (
```

Comment:
should we duplicate these tests for the one based on the CLI as well ?

Reply:
I duplicated just the part of [test](https://github.com/huggingface/datasets-server/pull/2463/files#diff-4da26c1315609343b7f12e022b53fe14f12ea1ce66eac62ad21995bb2f1ff144R32) computing